release/models/network-instance/openconfig-network-instance-l3.yang

```diff
@@ -134,6 +140,16 @@ module openconfig-network-instance-l3 {
         "The address families that are to be enabled for this
         network instance.";
     }
+
+    leaf fallback-network-instance {
+      type network-instance-ref;
+      description
+        "When this leaf is populated, the specified network instance
+        should be used as a fallback instance when a route cannot
```

Comment:
nit: how about "when a route lookup does not match anything in the local network-instance"?

Reply:
ping @robshakir